spec/features/requests/request_spec.rb

```diff
@@ -230,7 +230,6 @@
           expect(page).to have_content 'Pick-up location: Firestone Library'
           expect { click_button 'Request Selected Items' }.to change { ActionMailer::Base.deliveries.count }.by(1)
           expect(page).to have_content 'Your request was submitted. Our library staff will review the request and contact you with any questions or updates.'
-          expect(page).not_to have_content 'Request submitted to BorrowDirect'
```

Comment:
I thought that this expectation was no longer very meaningful, if there is no longer a way for the application to return this string.

Reply:
agreed